Mixer.UnitTests/ChatClientUnitTests.cs

```diff
@@ -172,6 +172,23 @@ public void ClearMessages()
             });
         }
 
+        [TestMethod]
```

Comment:
The test here is:
* Clear chat
* Get history 50 should return 0.
* Add 1 chat message
* Get history 50 should return 1.